docs/workforce-management-framework/topics/con-logger-module.adoc

```diff
@@ -0,0 +1,11 @@
+[id='con-logger-module-{chapter}']
+= Introducing the {WFM-RC-NameShort} Logger Module
+
+The *{WFM-RC-NameLong} Logger Module* is a logging facade for link:https://nodejs.org[Node.js] based applications and it is used by all {WFM-RC-NameLong} modules.
+The *{WFM-RC-NameLong} Logger Module* is contained in the `{WFM-RC-NameLong}-core` module.
+
+Developers can use their own loggers by wrapping their implementation into the provided _logger interfaces_.
+For more information about the _logger interface_ structure, see link:{WFM-RC-CoreURL}{WFM-RC-Branch}/cloud/logger/src/Logger.ts[Logger.ts] 
+
+The {WFM-RC-NameLong} Demo application contains two implementations of the _logger interface_ and are found in the `{WFM-RC-NameLong}-core` module.
+To view an example, see the link:{WFM-RC-CoreURL}{WFM-RC-Branch}/cloud/logger/example/index.ts[Cloud Logger] example.
```

Comment:
Do we have a Client logging example?

Reply:
logger has moved to common directory since this was written. will need to update.